src/providers/whatsapp/whatsapp.service.ts

```diff
@@ -24,6 +23,7 @@ export class WhatsappService {
       logQR: true,
       catchQR: this.onWaitQrCode,
       statusFind: this.onGetStatus,
+      autoClose: 90000,
```

Comment:
o que esse parametro faz?

Reply:
chega uma hora que o venom encerra a sessão por não ter sido lido o qrcode, esse é o tempo que ele leva encerrar

Reply:
nao tem como deixar pra sempre? ou infinito?

Reply:
Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false), ai fica infinito

Reply:
bora testar deixar infinito

Reply:
ok